### DISCUSSION TITLE
LLM-плагины для Obsidian

### DISCUSSION THREAD

**Discussion by itbeard** [it-beard/evocoders-usecases#12] [Obsidian]
Плагины из воркшопа по Obsidian с Machine Learning Engineer Игорем Нестереней (лежит [тут](https://t.me/c/2069889012/10/5618)):

🔹 text generator - автокомплит
🔹 ollama - интеграция ollama
🔹 copilot auto completion
🔹 auto classifier - автолинковка статей
🔹 smart connections 
🔹 copilot
🔹 Khoj - RAG по волту
🔹 view count - плагин частоты посещения заметок и тренды
🔹 Smart Second Brain

**Comment by lewocki** [comment]
Привет @itbeard
Это опечатка в названии?) Наверное раскладка переключилась ))
